fargate/upload-move/pkg/createClient.go

```diff
@@ -0,0 +1,38 @@
+package pkg
+
+import (
+	"context"
+	"errors"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	log "github.com/sirupsen/logrus"
+	"strings"
+)
+
+// CreateClient creates a client for the appropriate region data is being copied to
+func CreateClient(storageBucket string) (*s3.Client, string, error) {
+
+	// Get region
+	region := GetRegion(storageBucket)
+	if region.RegionCode == "" {
```

Comment:
minor nits: I think the Go pattern would be to have GetRegion return AWSRegsions, bool where bool is "exists" like the map get function e.g. `region, exists := Regions[shortname]` and use the `exists` in the if statement instead of the "empty" value of a string